dist_run.py

```diff
@@ -212,9 +315,8 @@ def main():
     logger.info(
         f"Stage {rank} has {color.blue}{stage_num_params} params{color.reset}, Size: {color.blue}{stage_size_formatted}{color.reset}\n"
     )
-    
+
     # Setup input position
-    # input_pos for prefill: a list of increasing integers from 0 to seqlen
```

Comment:
Is there a reason for removing this comment?

Reply:
ah sorry, reverted.  I thought metamate had added this (since I have to fight it from randomly adding things when I don't intend for it to) and didn't think two lines of comments was needed here. 
Let's collapse it though as we don't need two lines of "setup input position and then # input_pos for prefill - can just be:
"Setup input_pos for prefill: a list of increasing..."